### ISSUE TITLE
Error: Unexpected response format - {"error": "option \"temperature\" must be of type float32"}

### ISSUE THREAD

**Issue by alexdoroga** [if-ai/ComfyUI-IF_AI_tools#94]
![image](https://github.com/user-attachments/assets/fbc4b398-de9a-4f84-8b74-d23f308fa81f)

**Comment by aqgasa** [comment]
Try to change "temperature" in the "IF Chat Prompt" node. In the image you have it at 0.0. Try changing it to 0.2.

From the Instructions:
"temperature control the randomness of the response  goes from 0-2 a lower value like 0.2 will give very strict output for creative prompts 0.7-1.2 might work well above that starts to become wild but experiment your own settings"


Also, you may want to change the "repeat_penalty" value. At 0.2, like you have now, the process got stuck for a long time and the output text just repeated the same word over and over again. Changing "repeat_penalty" to 1.0 fixed that. You may try different values.

**Comment by if-ai** [comment]
Yes like aqgasa says recommended 0.7 but sometimes the node draws wrong values and have to be tweak. You have another values weed on NaN and Idky I have to update the node to make it compatible with latest front end that just came out so Idk if this is because you have the new ui

**Comment by alexdoroga** [comment]
> Yes like aqgasa says recommended 0.7 but sometimes the node draws wrong values and have to be tweak. You have another values weed on NaN and Idky I have to update the node to make it compatible with latest front end that just came out so Idk if this is because you have the new ui

... and with your settings it.s again dont work ))) 
![image](https://github.com/user-attachments/assets/34900d55-3e97-4f3a-a3e6-a659af38e37e)

**Comment by alexdoroga** [comment]
> Try changing it to 0.2.

Thank you, at least I moved from a dead point))

**Comment by if-ai** [comment]
You probably didn't do step 3 move the IF_AI folder from ComfyUI/custom_nodes/ComfyUI-IF_AI_tools to the root input with the rest of the presets.

When you select seed the temperature does not affect the generation only the seed number that is why there is a ramdomness mode button when you chose off the button it takes temperature mode. Also there is NaN value on the seed that should be a random number or fixed depending on your selections.

I think for that model you need a Llama 2 or 3 stopping string and probably 256 as max tokens. There is one wf that has help notes about each parameter included in the workflow folder

**Comment by if-ai** [comment]
Also the prompt you using is too large for an SD prompt I mean you can use a long prompt but it defeats the purpose of the tool the idea is that you give it something like small two or three words phrase and let the Ai create something with it
